forum/qa-content/qa-question.js

```diff
@@ -352,7 +339,7 @@ function handleRefusedPost({ submitButton, postRefuseReason, postType, parentId,
 
 	function showSubmissionError() {
 		const postSubmissionError = document.createElement('div');
-		postSubmissionError.innerHTML = getSubmissionErrorContent();
+		postSubmissionError.innerHTML = postRefuseReason || 'Wystąpił problem - nie udało się dodać posta.';

```

Comment:
Jeśli w zmiennej `postRefuseReason` jest sam tekst (bez kodu HTML), to `.innerHTML` lepiej zamienić na `.textContent`.

Reply:
ok, zamieniłem